pytorch_lightning/strategies/ddp_spawn.py

```diff
@@ -372,6 +369,11 @@ def register_strategies(cls, strategy_registry: Dict) -> None:
             description="DDPSpawn Strategy with `find_unused_parameters` as False",
             find_unused_parameters=False,
         )
+        strategy_registry.register(
```

Comment:
The strategy registry addition could be a separate PR

Reply:
Not really. The current accel_conn will have bunch of issues, I don't think it worse our time to fix the existing accel_conn
For example, the return from [this](https://github.com/PyTorchLightning/pytorch-lightning/blob/a64438c89717e4115d0f7cd4e8f93a812575a704/pytorch_lightning/trainer/connectors/accelerator_connector.py#L598) will be different. To make it work, we will need to fix the existing accel_conn

Reply:
I'm ok leaving it like this in the PR.